src/Symfony/Component/Messenger/Transport/RedisExt/Connection.php

```diff
@@ -203,24 +233,43 @@ public function reject(string $id): void
         }
     }
 
-    public function add(string $body, array $headers): void
+    /**
+     * @param int $delay The delay in milliseconds
```

Comment:
please call the param `$delayInMs` like in amqp connection. then you don't need this phpdoc

Reply:
@Tobion thank you for your review. Did change this!